packages/react-instantsearch/src/components/SearchBox.js

```diff
@@ -203,9 +203,11 @@ class SearchBox extends Component {
     const { translate, autoFocus } = this.props;
     const query = this.getQuery();
 
-    const submitComponent = this.props.submitComponent;
-    const resetComponent = this.props.resetComponent;
-    const loadingIndicatorComponent = this.props.loadingIndicatorComponent;
+    const {
+      loadingIndicatorComponent,
+      resetComponent,
+      submitComponent,
+    } = this.props;
```

Comment:
We could merge the destructuring props with the one above. Because we destruct the same attributes two times (see line `203`).